### DISCUSSION TITLE
Pressing space bar when viewing a file toggles symbols instead of scrolling

### DISCUSSION THREAD

**Discussion by raphaelrk** [community/community#46520] [Accessibility]
### Select Topic Area

Bug

### Body

When you go to a file it automatically highlights the "x" in Symbols, so then when your press space instead of scrolling down it just toggles symbols open/closed

I imagine the initial focus on symbols is for accessibility, which makes sense. But when you press space it should scroll, not toggle symbols open/closed.

Workaround for the keyboard lovers out there is to press shift+tab, then space. But shift+tab also does a weird scroll past the start of the file

Image of the "X" auto-highlighted when visiting [https://github.com/geohot/minikeyvalue/blob/master/src/lib.go](https://github.com/geohot/minikeyvalue/blob/master/src/lib.go):

<img width="1341" alt="image" src="https://user-images.githubusercontent.com/3377845/217011244-9569b122-b51e-4aad-ac74-eacfdabd658d.png">

Immediately after pressing shift+tab:

<img width="1663" alt="image" src="https://user-images.githubusercontent.com/3377845/217012893-11bc614b-fc13-42c8-8c2c-5fe2e33c6864.png">

**Comment by cehfisher** [comment]
Hi @raphaelrk - we wanted to follow up on this reported bug. We were unable to replicate this issue in our review. Is this an issue you can replicate currently? If the original issue is resolved, could you please close this out? If it is not resolved, could you supply more detailed instructions to reproduce your issue and include screenshots and/or URLs, if applicable. Thank you! (cc @queenofcorgis)
